share/config_files/common/rsyslog.conf

```diff
@@ -15,35 +32,59 @@ $InputFileStateFile anaconda-tb-statefile
 $InputFileFacility local1
 $InputRunFileMonitor
 
+
 #### GLOBAL DIRECTIVES ####
+
+# Where to place auxiliary files
+#$WorkDirectory /var/lib/rsyslog
+
 # Use default timestamp format
 $ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat
 
+# File syncing capability is disabled by default. This feature is usually not required,
+# not useful and an extreme performance hit
+#$ActionFileEnableSync on
```

Comment:
There is no change regarding this option.
I'd leave it for follow-up consideration.